EZSwiftExtensionsTests/UIFontTests.swift

```diff
@@ -13,7 +13,7 @@ class UIFontTests: XCTestCase {
     
     let TEST_SIZE = CGFloat(10)
     
-    let TEST_FONT_TYPE = FontType.Bold
+    let TEST_FONT_TYPE = FontType.bold
```

Comment:
Yea I guess you can. Its essentially refactoring, feel free to update the PR. I am approving this for now.